README.md

```diff
@@ -198,18 +198,18 @@ The latest upstream code can be obtained from GitHub:
 ## Licensing
 
 VDSM is provided under the terms of the GNU General Public License,
-version 2 or later. Please see the COPYING file for complete GPLv2+
+version 2 or later. Please see the `COPYING` file for complete GPLv2+
 license terms.
 
 In addition, as a special exception, Red Hat, Inc. and its affiliates
 give you permission to distribute this program, or a work based on it,
 linked or combined with the OpenSSL project's OpenSSL library (or a
 modified version of that library) to the extent that the library, or
 modified version, is covered by the terms of the OpenSSL or SSLeay
-licenses.  Corresponding source code for the object code form of such
+licenses. Corresponding source code for the object code form of such
 a combination shall include source code for the parts of OpenSSL
 contained in the combination.
 
 If you modify this program, you may extend this exception to your
-version, but you are not obligated to do so.  If you do not wish to do
+version, but you are not obligated to do so. If you do not wish to do
 so, delete this exception statement from your version.
```

Comment:
Removing the 2 spaces at the end is not needed. Quoting names in the entire document looks nice.